R/gitlab.R

```diff
@@ -44,39 +44,20 @@ gitlabDownloadImpl <- function(url, destfile, ...) {
   return(TRUE)
 }
 
-#' Retrieve GitLab user.
-#'
-#' A GitLab user
-#' Looks in env var \code{GITLAB_USERNAME}
-#'
-#' @keywords internal
-#'
-gitlab_user <- function(quiet = FALSE) {
-  user <- Sys.getenv("GITLAB_USERNAME")
-  if (nzchar(user)) {
-    if (!quiet) {
-      message("Using GitLab username from envvar GITLAB_USERNAME")
-    }
-    return(user)
-  }
-  return(NULL)
-}
-
-
-#' Retrieve GitLab password
+#' Retrieve GitLab PAT.
 #'
-#' A GitLab password
-#' Looks in env var \code{GITLAB_PASSWORD}
+#' A GitLab PAT
+#' Looks in env var \code{GITLAB_PAT}
 #'
 #' @keywords internal
 #'
-gitlab_pwd <- function(quiet = FALSE) {
-  pwd <- Sys.getenv("GITLAB_PASSWORD")
-  if (nzchar(pwd)) {
+gitlab_pat <- function(quiet = FALSE) {
```

Comment:
I just realized that the bitbucket and gitlab accessor functions are exported by packrat. I don't think that it was our intent to allow `packrat::gitlab_pat`. That said, it's out in the wild. :(

CC @kevinushey

Reply:
I don't think they are actually exported! They don't have the `@export` keyword, they don't appear in `NAMESPACE`, and you need to use `:::` to access them.

Reply:
They do have documentation generated, but it isn't included in the index, because they include `@keyword internal` in a `roxygen2` comment.